docs/modules/develop/pages/guide_migrate_webpacker_app.adoc

```diff
@@ -35,13 +35,24 @@ bundle install
 bundle exec rails webpacker:install
 ----
 
-3. Install npm dependencies
+3. Install Decidim webpacker custom code
 
 [source,console]
 ----
-npm install
+bundle exec rails decidim:webpacker:install
 ----
 
+This task do a few steps to allow the Rails instance to have a webpacker instance sharing the code between itself and Decidim gem.
+
+Steps it does:
+
+- copies babel and postcss configuration to instance application
+- copies config/webpacker.yml configuration to instance application
```

Comment:
We should remove this after merging #8066, right?

Reply:
Yes, fixed